playbooks/openshift-logging/private/config.yml

```diff
@@ -13,6 +13,13 @@
           status: "In Progress"
           start: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
 
+- name: Convert openshift_logging_es_nodeselector using map_to_pairs
```

Comment:
We should remain consistent with other usages of `map_*`. Please remove this and add the `map_to_pairs` to where `openshift_logging_es_nodeselector` is used.